[LLM] Add vllm instruction for other clouds (#2445)
skypilot-org/skypilot: llm/vllm/README.md

1. Start serving the Llama-2 model:
```bash
sky launch -c vllm-llama2 serve-openai-api.yaml
```

> **Optional**: You can also try to use other GPUs by specifying the `--gpus` flag to enable other clouds (The original L4 only presents on GCP). For example, to use V100 GPUs:

Reviewer: ```suggestion
**Optional**: Only GCP offers the specified L4 GPUs currently. To use other clouds, use the `--gpus` flag to request other GPUs. For example, to use V100 GPUs:
```
